torch/testing/_internal/common_methods_invocations.py

```diff
@@ -6682,7 +6682,7 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            autodiff_nonfusible_nodes=["aten::hardswish"]),
     OpInfo('nn.functional.unfold',
            aten_name='im2col',
-           dtypes=floating_types_and(torch.half),
+           dtypes=floating_types_and(torch.half, torch.bfloat16),
```

Comment:
You'll want to separate this into dtypesIfCPU and dtypesIfCUDA: https://github.com/pytorch/pytorch/blob/9573e7a644544fbd623e652766d47c3e84d588de/torch/testing/_internal/common_methods_invocations.py#L4860-L4861

Reply:
I separated this into dtypesIfCPU. Thank you.